tests/e2e_regression/e2e_config.yml

```diff
@@ -2,23 +2,24 @@ analysis:
   source:
     cache:
       cache_dir: tests/e2e_regression/data
```

Comment:
can we pls rename `data` to `rpc-cache` (similar to [what we have in k8s](https://github.com/oasislabs/private-charts/blob/14b978e81ccf2928e0ac2b0188f290c1cf61e558/charts/oasis-indexer/templates/deployment.yaml#L72-L72)), or else something more informative?

Reply:
sure, done